css/main.css

```diff
@@ -1,3 +1,213 @@
 /* Add your CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+
+header {
+  display: grid;
+  grid-template-columns: 20px 1fr 60px;
+  grid-template-rows: 50px;
+  align-items: center;
+  column-gap: 10px;
+  margin: 20px 15px 0 15px;
+  font-family: "Poppins", sans-serif;
+  color: black;
+  line-height: 2%;
+}
+
+main {
+  font-family: "Quicksand", sans-serif;
+  display: flex;
```

Comment:
Remember, if you want elements to just stack vertically, block elements like `div` or `main` will do it naturally without any CSS, there's no need to use `flex`